topics/generic-metrics-sets-subscription-results.yaml

```diff
@@ -0,0 +1,9 @@
+topic: generic-metrics-sets-subscription-results
```

Comment:
This sets/distributions/counters split of generic metrics doesn't seem right to me. I'd like to get that changed. Wanna just omit this from here for now? We'll try and fix that then add it later?

Reply:
Sure I can take them out 👍